zebra-chain/src/serialization/read_zcash.rs

```diff
@@ -17,12 +20,17 @@ pub trait ReadZcashExt: io::Read {
     ///
     /// Preallocating vectors using untrusted `CompactSize`s allows memory
     /// denial of service attacks. Valid sizes must be less than
-    /// `MAX_BLOCK_BYTES / min_serialized_item_bytes` (or a lower limit
+    /// `MAX_PROTOCOL_MESSAGE_LEN / min_serialized_item_bytes` (or a lower limit
     /// specified by the Zcash consensus rules or Bitcoin network protocol).
     ///
+    /// As a defence-in-depth for memory preallocation attacks,
```

Comment:
```suggestion
    /// As a defense-in-depth for memory preallocation attacks,
```

Reply:
Oh man, I just realized this is a US-English convention. Removing.

Reply:
I don't mind either way